lean/commands/data/generate.py

```diff
@@ -12,15 +12,28 @@
 # limitations under the License.
 
 from datetime import datetime
-from typing import Optional
+from typing import List, Optional
 
-from click import command, option, Choice, IntRange
+from click import command, option, Choice, IntRange, BadParameter
+import re
 
 from lean.click import DateParameter, LeanCommand
 from lean.constants import DEFAULT_ENGINE_IMAGE
 from lean.container import container
 
 
+def ticker_list_check(ctx, param, tickers_csv: str) -> List[str]:
+    if tickers_csv == "":
+        return []
+    tickers = tickers_csv.split(",")
+    for sym in tickers:
+        if not re.match("[A-Z]{1,5}", sym):
+            raise BadParameter(
+                f"need comma separated ALLCAPS symbol(s), not '{sym}'"
```

Comment:
Suggest avoiding these specific validations, just passing it over and leaving it up to the core implementation to throw if required. For example do not think it's required for the tickers to be upper case

nit - ctx/param arguments aren't used, should be removed  👍

Reply:
can entirely remove the callback to let toolbox.dll figure out whats wrong, but now am just curious. I think ctx and param are what click sends along with the option values when the callback is called. is there an idiomatic way to avoid that or denote they're unused. (https://stackoverflow.com/questions/10025680/how-can-i-denote-unused-function-arguments suggests `del ctx param`. my linter didn't like `def func(_,_,val):`


Thanks for the ridiculously generous review of very small changes (and lots of useless code!)